alda-mode.el

```diff
@@ -117,14 +117,14 @@ ARGUMENT TEXT The text to play with the current alda server."
 ;;     (message "no mark was set")
 ;;     (alda-append-text (buffer-substring-no-properties start end))))
 
-;; (defun alda-play-region (start end)
-;;   "Plays the current selection in alda.
-;; Argument START The start of the selection to play from.
-;; Argument END The end of the selection to play from."
-;;   (interactive "r")
-;;   (if (eq start end)
-;;     (message "No mark was set!")
-;;     (alda-play-text (buffer-substring-no-properties start end))))
+(defun alda-play-region (start end)
```

Comment:
Whoops, I think I was a little aggressive in cutting out the `alda append` stuff above. Thanks! :+1: